cmd/get/launch_plan.go

```diff
@@ -0,0 +1,70 @@
+package get
+
+import (
+	"context"
+	"encoding/json"
+	"github.com/lyft/flytectl/cmd/config"
+	cmdCore "github.com/lyft/flytectl/cmd/core"
+	"github.com/lyft/flytectl/pkg/printer"
+	"github.com/lyft/flyteidl/gen/pb-go/flyteidl/admin"
+	"github.com/lyft/flytestdlib/logger"
+)
+
+// PrintableLaunchPlan is the structure for printing Launch Plan
+type PrintableLaunchPlan struct {
+	Version          string `header:"Version"`
+	Name             string `header:"Name"`
+	Type             string `header:"Type"`
+	Discoverable     bool   `header:"Discoverable"`
+	DiscoveryVersion string `header:"DiscoveryVersion"`
+}
+
+var launchPlanStructure = map[string]string{
+	"Version":          "$.id.version",
+	"Name":             "$.id.name",
+	"Type":             "$.closure.compiledTask.template.type",
+	"Discoverable":     "$.closure.compiledTask.template.metadata.discoverable",
+	"DiscoveryVersion": "$.closure.compiledTask.template.metadata.discovery_version",
+}
+
+func transformLaunchPlan(jsonbody []byte) (interface{}, error) {
+	results := PrintableLaunchPlan{}
+	if err := json.Unmarshal(jsonbody, &results); err != nil {
+		return results, err
+	}
+	return results, nil
+}
+
+func getLaunchPlanFunc(ctx context.Context, args []string, cmdCtx cmdCore.CommandContext) error {
+	launchPlanPrinter := printer.Printer{}
+
+	launchPlans, err := cmdCtx.AdminClient().ListLaunchPlans(ctx, &admin.ResourceListRequest{
```

Comment:
can you use the new NamedId lister i created in adminutils

Reply:
I dont see this change? This is an example of what i mean
https://github.com/lyft/flytectl/blob/master/cmd/get/task.go#L62

Reply:
I think if you follow the set precedence then you can greatly simplify the code